projects/packages/videopress/src/client/utils/video-chapters/validate-chapters/index.ts

```diff
@@ -0,0 +1,53 @@
+import type { VideoPressChapter } from '../types';
+
+/**
+ * Calculates the number of seconds from a timestamp in the format hh:mm:ss
+ *
+ * @param {string} time - The timestamp
+ * @returns {number}    - The number of seconds
+ */
+function getSeconds( time: string ): number {
```

Comment:
We probably need to change this to be able to support the `mm:ss` format (see comment below).

Reply:
Never mind, comment below explains why this is not necessary.